test/mysql-mariadb.spec.js

```diff
@@ -491,6 +491,20 @@ describe('mysql', () => {
           "SELECT (SELECT GROUP_CONCAT(`v`) FROM `keyperson` WHERE `e`.`keyperson` REGEXP concat('\b', `k`, '\b')) AS `keyperson` FROM `abc` AS `e`"
         ]
       },
+      {
+        title: 'set op INTERSECT',
+        sql: [
+          `SELECT * FROM (SELECT 1) INTERSECT SELECT * FROM (SELECT 2)`,
```

Comment:
```suggestion
          `SELECT * FROM (SELECT 1) intersect SELECT * FROM (SELECT 2)`,
```

Should we use lowercase keyword for test?

Reply:
it's case-insensitive.

Reply:
updated